Src/Common/Domain/interface/IVolunteerOpportunity.cs

```diff
@@ -1,18 +1,29 @@
 ﻿using System;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace Ttu.Domain
 {
     public interface IVolunteerOpportunity
     {
 
         // attributes
-        IUser CreatedBy { get; }
+
+        // Who created this project

```

Comment:
Probably best without the specific comment. The (// attributes and // behavior) comments are there to separate attributes from behavior since regions tend to be a big heavyweight for an interface (since there are no implementations).